WordPress/twentytwentyfive
# Move the color palette presets to a subfolder under styles

**Description**
The color palette presets are placed directly in the styles folder.
Once the "combined" style variation files are added, the number of JSON files in the styles folder will be doubled.
This PR moves the color palette presets to a separate folder, to organize the files and make the long term maintenance of the JSON files easier.

Partial for: https://github.com/WordPress/twentytwentyfive/issues/193

**Testing Instructions**
Go to Appearance > Editor > Styles and confirm that the palette presets are still available in the Site Editor.
